src/ngx_http_lua_ssl.h

```diff
@@ -26,6 +26,7 @@ typedef struct {
 
     int                      exit_code;  /* exit code for openssl's
                                             set_cert_cb callback */
+    int                      http_version;
```

Comment:
You just want to allow 1 and 2 here? Better be explicit. We do have protocol versions like 0.9, 1.0, and 1.1 also in the nginx core. If you just want to signify whether it is HTTP/2, then we should just make this a boolean field with a name like `is_http2`. This way, we can also make this field just take 1 bit.